Cargo.lock

```diff
@@ -1309,17 +1289,17 @@ dependencies = [
 
 [[package]]
 name = "ring"
-version = "0.16.20"
+version = "0.17.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3053cf52e236a3ed746dfc745aa9cacf1b791d846bdaf412f60a8d7d6e17c8fc"
+checksum = "c17fa4cb658e3583423e915b9f3acc01cceaee1860e33d59ebae66adc3a2dc0d"
 dependencies = [
  "cc",
+ "cfg-if",
+ "getrandom",
```

Comment:
Hmm, seems that *ring* is now pulling in `getrandom` which is breaking WASM builds.

Reply:
I removed the lockfile update, as it didn't directly relate to upgrading tonic.